ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/verticaltabs/v1/verticaltabs/clientlibs/contentframe/css/tabsview.css

```diff
@@ -0,0 +1,38 @@
+.cmp-verticaltabs{
+
+}
+.cmp-verticaltabs__label{
+
+}
+.cmp-verticaltabs__label-container{
+  
+}
+.cmp-verticaltabs__tabs-container {
+}
+
+.cmp-verticaltabs__tablist{
+}
+
+.cmp-verticaltabs__tab{
+}
+
+.cmp-verticaltabs__tab.cmp-verticaltabs__tab--active{
+
+}
+
+.cmp-verticaltabs__tabpanel{
+  display: none;
+}
+
+.cmp-verticaltabs__tabpanel.cmp-verticaltabs__tabpanel--active{
```

Comment:
Why would an element have both of these classes at the same time?

Reply:
`cmp-verticaltabs__tabpanel`  :  for styling of all tabpanels and by default all tabpanels are hidden.
`cmp-verticaltabs__tabpanel--active` active tab panel is assigned with this class that only governs visibility styole